features/users.feature

```diff
@@ -0,0 +1,665 @@
+          "projectsCount": 0,
+          "membershipsCount": 2,
+          "createdAt": "@iso8601",
+          "updatedAt": "@iso8601"
+        }],
+        "active": true,
+        "roles": [],
+        "createdAt": "@iso8601"
+      }, {
+        "id": "@idOf: lskywalker",
+        "name": "lskywalker",
+        "technical": false,
+        "primaryEmailMd5": "@md5",
+        "primaryEmail": "luke.skywalker@localhost.localdomain",
+        "emails": [{
+          "address": "luke.skywalker@localhost.localdomain",
+          "active": false
+        }],
+        "organizations": [{
+          "id": "@idOf: Rebel Alliance",
+          "name": "rebel-alliance",
+          "public": false,
+          "displayName": "Rebel Alliance",
+          "projectsCount": 0,
+          "membershipsCount": 2,
+          "createdAt": "@iso8601",
+          "updatedAt": "@iso8601"
+        }],
+        "active": false,
+        "roles": [],
+        "createdAt": "@iso8601"
+      }, {
+        "id": "@idOf: master",
+        "name": "master",
+        "technical": false,
+        "primaryEmailMd5": "@md5",
+        "primaryEmail": "@email",
+        "emails": [{
+          "address": "@email",
+          "active": false
+        }],
+        "organizations": [],
+        "active": true,
+        "roles": [
+          "admin"
+        ],
+        "createdAt": "@iso8601"
+      }, {
+        "id": "@idOf: palpatine",
+        "name": "palpatine",
+        "technical": false,
+        "primaryEmailMd5": "@md5",
+        "primaryEmail": "palpatine@localhost.localdomain",
+        "emails": [{
+          "address": "palpatine@localhost.localdomain",
+          "active": false
+        }],
+        "organizations": [{
+          "id": "@idOf: Galactic Empire",
+          "name": "galactic-empire",
+          "public": true,
+          "displayName": "Galactic Empire",
+          "projectsCount": 0,
+          "membershipsCount": 3,
+          "createdAt": "@iso8601",
+          "updatedAt": "@iso8601"
+        }],
+        "active": true,
+        "roles": [],
+        "createdAt": "@iso8601"
+      }, {
+        "id": "@idOf: pamidala",
+        "name": "pamidala",
+        "technical": false,
+        "primaryEmailMd5": "@md5",
+        "primaryEmail": "padme.amidala@localhost.localdomain",
+        "emails": [{
+          "address": "padme.amidala@localhost.localdomain",
+          "active": false
+        }],
+        "organizations": [{
+          "id": "@idOf: Old Republic",
+          "name": "old-republic",
+          "public": true,
+          "displayName": "Old Republic",
+          "projectsCount": 0,
+          "membershipsCount": 3,
+          "createdAt": "@iso8601",
+          "updatedAt": "@iso8601"
+        }],
+        "active": true,
+        "roles": [],
+        "createdAt": "@iso8601"
+      }]
+      """
+    And nothing should have been added or deleted
+
+
+
+  Scenario: An organization member should be able to retrieve a user by its email.
```

Comment:
The naming of this scenario and the following ones is not consistent. I would suggest this (in the current order):

* An organization member should be able to retrieve a user by e-mail
* An organization member should be able to search a user by name
* An organization member should be able to search a user by e-mail
* An organization member should be able to retrieve a user by name